src/main/thrift/tag.thrift

```diff
@@ -103,6 +104,11 @@ struct PaidContentInformation {
     2: optional string campaignColour;
 }
 
+struct CampaignInformation {
+    /** The type of campaign tag that this is, e.g. Callout */
+    1: required string campaignType;
```

Comment:
Generally if it's a 'type' field we define an enum rather than using a string. I'm not sure what else this field will be used for. @currysoup what do you think?

Reply:
As someone new to the system I'm going to err on the side of simplicity by using the existing precedent. Can note this as an issue needing refactoring on the repo if someone else is feeling more energetic?

Reply:
ok, let's stick with the string then :)